### PR TITLE
Add k8s.io/code-generator dependency via dep

### PR DESCRIPTION
**Pull request by mestery**
Signed-off-by: Kyle Mestery <mestery@mestery.com>

### CODE REVIEW

**Comment by mestery** [comment]
The idea is to merge the generators as this separate patch, then I can rebase #54 and make that one a bit smaller.

**Comment by mestery** [comment]
I should note this PR is ultimately a part of #48.

File: Gopkg.toml [hunk 1]
```diff
@@ -55,10 +61,20 @@ required = ["github.com/golang/protobuf/protoc-gen-go"]
...
   version = "3.3.5"
 
 [prune]
   go-tests = true
   unused-packages = true
+
+  [[prune.project]]
+    name = "k8s.io/code-generator"
+    unused-packages = false
+    non-go = false
```
Comment (fkautz) [inline Gopkg.toml]: non-go = false is redundant and should be removed.

Comment (mestery) [reply Gopkg.toml]: I was wondering about this, thanks!

**Comment by mestery** [comment]
Verified k8s.io/gengo and k8s.io/code-generator are both licensed under Apache 2. golang.org/x/tools uses the following license:

https://github.com/golang/tools/blob/master/LICENSE

**Pull request merged**